pandas/core/indexes/datetimes.py

```diff
@@ -2231,7 +2231,7 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
 
```

Comment:
Can you replace `string` by `str` please?

Reply:
Yes I will do that correction.